packages/liveblocks-client/src/room.test.ts

```diff
@@ -87,70 +80,14 @@ describe("room / auth", () => {
     consoleErrorSpy.mockRestore();
   });
 
-  test("should reuse token after reconnect", async () => {
```

Comment:
The two following tests are not validating what they think they're validating (and never have) for a few reasons.

`withDateNow` does not accept a promise as a callback.
`withDateNow` expects a number in millisecond and not in second.
`withDateNow` breaks `waitFor` because it use `Date.now` to retry.

We should use the state machine to have a better control of time to really validate token reuse. I'll create another ticket just to reintroduce those tests properly.

Reply:
> `withDateNow` does not accept a promise as a callback.
> `withDateNow` expects a number in millisecond and not in second.
> `withDateNow` breaks `waitFor` because it use `Date.now` to retry.

Woah! 🤯 Thank you for digging in here!